### DISCUSSION TITLE
qmake from qt generating invalid Makefile

### DISCUSSION THREAD

**Discussion by Ndolam** [Homebrew/discussions#152] [Everyday usage]
I'm guessing some recent software update that Apple pushed on me caused this.  Still, people should know.

qmake creates lines like	/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -pipe -stdlib=libc++ -O2 -std=gnu++11 $(EXPORT_ARCH_ARGS) -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -mmacosx-version-min=10.13 -Wall -Wextra -dM -E -o moc_predefs.h /usr/local/Cellar/qt/5.15.1/mkspecs/features/data/dummy.cpp

but there is no longer a directory "MacOSX10.15.sdk" on my mac.  It is now just MacOSX.sdk.
Trying to compile with qmake's Makefile crashes and burns.  Editing the Makefile (e.g.)
:g/MacOSX10.15.sdk/s//MacOSX.sdk/g
did the trick for me.

Should qt be updated to reflect this change?  Or is it just me whose directory structure mysteriously changed?

Cheers.

**Comment by SMillerDev** [comment]
I guess that should be patched in CMake then
